Frontend/src/components/TheNavbar.vue

```diff
@@ -26,7 +26,10 @@
             <ul class="dropdown-menu">
               <li>
                 <router-link
-                  :to="{ name: 'FlightsAll', params: { year: currentYear } }"
+                  :to="{
+                    name: 'FlightsAllYear',
```

Comment:
Ich würde FlightsOfSeason vorschlagen. (Bis mir was besseres einfällt:D)

Reply:
Warum überhaupt das rename? Weil es nicht alle überhaupt sind?

Reply:
Weil in der Navbar immer die Seite mit den Flügen des aktuellen Jahres geladen werden soll (so habe ich das zumindestens aus deiner Implementierung herausgelesen).

Wenn ich aber in FlightsAll "year" als Parameter drin lasse, muss ich das auch jedes mal mitgeben, wenn ich die Seite öffnen will. Dann filtert er natürlich dann auch nach Jahr, was ich aber nicht unbedingt will.

Da die Identifizierung im router über den Namen läuft, habe ich deshalb einen Link mit und eine ohne Verpflichtung zum Year-Prop gemacht.

Reply:
AH verstehe was du meinst, hab noch ne elegantere Lösung im Kopf. Versuche ich demnächst mal.